src/autopas/AutoPas.h

```diff
@@ -672,7 +673,7 @@ class AutoPas {
   /**
    * Specifies after how many pair-wise traversals the neighbor lists are to be rebuild.
    */
-  unsigned int _verletRebuildFrequency{20};
+  unsigned int _verletRebuildFrequency{18};
```

Comment:
why change this?
If you want the warning not to spawn with default parameters, `_numSamples` has to be a multiple of `_verletRebuildFrequency` not vice-versa.

Reply:
reverted